<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Taschenrechner</class>
 <widget class="QMainWindow" name="Taschenrechner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Taschenrechner</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>func</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="add">
       <property name="text">
        <string>plus</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="min">
       <property name="text">
        <string>minus</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="div">
       <property name="text">
        <string>durch</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mult">
       <property name="text">
        <string>mal</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ausgabe</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>71</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>zahlensystem</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QRadioButton" name="dec">
       <property name="text">
        <string>dec</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="hex">
       <property name="text">
        <string>hex</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="oct">
       <property name="text">
        <string>oct</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="bin">
       <property name="text">
        <string>bin</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>19</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>261</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="num1"/>
     </item>
     <item>
      <widget class="QLineEdit" name="num2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>82</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="operation">
       <property name="text">
        <string>berechne</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ende">
       <property name="text">
        <string>beende</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>ende</zorder>
   <zorder>operation</zorder>
   <zorder>num1</zorder>
   <zorder>num2</zorder>
   <zorder>groupBox</zorder>
   <zorder>label</zorder>
   <zorder>operation</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>lcdNumber</zorder>
   <zorder>reset</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>num1</tabstop>
  <tabstop>num2</tabstop>
  <tabstop>add</tabstop>
  <tabstop>min</tabstop>
  <tabstop>div</tabstop>
  <tabstop>mult</tabstop>
  <tabstop>operation</tabstop>
  <tabstop>ende</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ende</sender>
   <signal>clicked()</signal>
   <receiver>Taschenrechner</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dec</sender>
   <signal>clicked()</signal>
   <receiver>lcdNumber</receiver>
   <slot>setDecMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hex</sender>
   <signal>clicked()</signal>
   <receiver>lcdNumber</receiver>
   <slot>setHexMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oct</sender>
   <signal>clicked()</signal>
   <receiver>lcdNumber</receiver>
   <slot>setOctMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bin</sender>
   <signal>clicked()</signal>
   <receiver>lcdNumber</receiver>
   <slot>setBinMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reset</sender>
   <signal>clicked()</signal>
   <receiver>num1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
